src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -128,7 +130,7 @@ ApiGlobe.prototype.addImageryLayerFromJSON = function addImageryLayerFromJSON(ur
 
 /**
  * This function adds an imagery layer to the scene using an array of JSON files. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
- * @constructor
+ * <iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/763trrk8/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>
```

Comment:
Nice! Can you put an explicit `https` scheme? It would allow to test locally, when we access the html with the `file:///` scheme